tests/expected/array/expected

```diff
@@ -4,5 +4,3 @@ SUCCESS\
 assertion failed: y[1] == 2
 FAILURE\
 index out of bounds: the length is less than or equal to the given index
-FAILURE\
-assertion failed: y[z] == 3
```

Comment:
What happened to this check

Reply:
Control flow stops at the previous assertion failure. The "check" is still there, but now succeeds. (In the case where it would fail, it fails on the earlier assertion, and it's not reported as "unreachable" because paths that would succeed for both checks still exist.)